### PR TITLE
Fix configuration test running every Chef run regardless of service action

### PR DESCRIPTION
**Pull request by bmhughes**
Description

- Fix configuration test running every Chef run regardless of service action

## Issues Resolved

- None

## Check List

- [x] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by bmhughes** [comment]
Just the same fix as implements in the DHCP cookbook a few days ago.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 1.0.1 (2020-05-20)
```
Comment (damacus) [inline CHANGELOG.md:3]:
```suggestion
## Unreleased
```

We have some magic scripts that take care of this now :)

File: metadata.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
 source_url        'https://github.com/sous-chefs/syslog_ng'
 issues_url        'https://github.com/sous-chefs/syslog_ng/issues'
 chef_version      '>= 14.0'
-version           '1.0.0'
+version           '1.0.1'
```
Comment (damacus) [inline metadata.rb:9]:
```suggestion
version           '1.0.0'
```

**Comment by damacus** [comment]
@bmhughes thanks for putting the release label on the PR, I now have a something to test our release pipeline with! 

Could you revert the version changes and I'll bump and release to supermarket with our new script. 

Thanks!

**Comment by bmhughes** [comment]
Ooh interesting, as good a chance as any to test. I'll get it done shortly 👍.

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: resources/service.rb [hunk 3]
```diff
@@ -22,22 +22,32 @@
...
 action_class do
   def do_service_action(resource_action)
-    if new_resource.config_test && %w(start restart reload).include?(resource_action)
-      command_path = platform_family?('amazon') ? '/sbin/syslog-ng' : '/usr/sbin/syslog-ng'
-      execute 'syslog-ng-config-test' do
-        command "#{command_path} -s"
-        action :run
-      end
-    end
+    log new_resource.config_test.to_s
```
Comment (damacus) [inline resources/service.rb:35]: We probably don’t want to leave the log resource in. Chef 15 users will have this trigger on every run

**Comment by bmhughes** [comment]
They shouldn't be in there still 🤦‍♂️, I was having trouble with it triggering and was using those for debug.

**Comment by damacus** [comment]
Ace, approved 👍 feel free to merge when its all done checking

**Pull request merged**